charts/rollout-operator/templates/webhook-clusterrole.yaml

```
@@ -0,0 +1,16 @@
+{{- if .Values.webhooks.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "rollout-operator.fullname" . }}-webhook-clusterrole
+rules:
+  - apiGroups:
+      - admissionregistration.k8s.io
+    resources:
+      - validatingwebhookconfigurations
+      - mutatingwebhookconfigurations
+    verbs:
+      - list
+      - patch
+      - watch
+  {{- end -}}
```

[nit] I'd align this to the starting `{{- if ...`

```suggestion
{{- end -}}
```

(this applies in a bunch of places)